test/instances/aave/AaveHyperdrive.t.sol

```diff
@@ -58,18 +58,40 @@ contract AaveHyperdriveTest is InstanceTest {
                 minimumShareReserves: 1e15,
                 minimumTransactionAmount: 1e15,
                 positionDuration: POSITION_DURATION,
+                fees: IHyperdrive.Fees({
+                    curve: 0,
+                    flat: 0,
+                    governanceLP: 0,
+                    governanceZombie: 0
+                }),
                 enableBaseDeposits: true,
                 enableShareDeposits: true,
                 enableBaseWithdraws: true,
                 enableShareWithdraws: true,
                 baseWithdrawError: new bytes(0),
                 isRebasing: true,
-                fees: IHyperdrive.Fees({
-                    curve: 0,
-                    flat: 0,
-                    governanceLP: 0,
-                    governanceZombie: 0
-                })
+                // The base test tolerances.
```

Comment:
I think using a builder pattern off a default configuration might make this more digestable by highlighting which config fields are out of the ordinary....

Maybe something like... 

```solidity
DefaultInstanceTestConfig
    .fees(0)
    .roundTripLongInstantaneousWithBaseUpperBoundTolerance(1e3)
    ...
```



Not something that needs to get done now, just food for thought

Reply:
Yeah, this could be nice. I don't know how well this would work considering that this is a constructor param, but worth playing around with in the future.